build.gradle

```diff
@@ -1,17 +1,14 @@
-import java.io.FileOutputStream
-import java.io.IOException
-import java.io.OutputStream
-import java.util.Properties
 
 apply plugin: 'groovy'
 apply plugin: 'maven-publish'
 apply plugin: 'signing'
 apply plugin: "com.gradle.plugin-publish"
 
-archivesBaseName = 'liberty-gradle-plugin'
 group = 'io.openliberty.tools'
 version = '3.8.4-SNAPSHOT'
-
+base {
```

Comment:
Do we need to apply the `base` plugin? The example [here](https://docs.gradle.org/8.8/userguide/upgrading_version_8.html#base_convention_deprecation) shows configuring that.

Reply:
its working even without adding base plugin explicitly. to avoid confusion, added apply plugin line for base plugin